src/averageAges.js

```diff
@@ -15,11 +15,18 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  let man = people.filter(p => p.sex === 'm');
```

Comment:
it's a set of `men` - not just one `man` )
```suggestion
  let men = people.filter(p => p.sex === 'm');
```